portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,9 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#title{
+
+    text-align:center;
```

Comment:
Unindent by 2 spaces so this line aligns with the rest of the nested blocks in the file. Eg. Align it with the margin-top: 20px; line in the block above. Also unindent the line below this one.